docs/messages_guide/src/asciidoc/_chapters/binder_msgs.adoc

```diff
@@ -1861,13 +1848,13 @@ incompatible data types.
 == SQL 4128
 
 ```
-Default volume and subvolume information could not be retrieved fromc=_DEFAULTS define - DEFINEINFO error <number>.
+Default volume and sub volume information could not be retrieved from=_DEFAULTS define - DEFINEINFO error <number>.
```

Comment:
I would think subvolume would be correct, however, this message is Guardian-specific and should never occur in Trafodion.